x/txfees/keeper/keeper.go

```diff
@@ -16,7 +16,7 @@ type Keeper struct {
 
 	accountKeeper       types.AccountKeeper
 	bankKeeper          types.BankKeeper
-	gammKeeper          types.GammKeeper
+	gammKeeper          types.SwapRouterKeeper
```

Comment:
is the plan to keep this as gammKeeper utilizing swaprouterkeeper, or does this eventually change?